queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT * from animals WHERE TO_CHAR(date_of_birth, 'yyyy') BETWEEN '2016' and '2019';
+SELECT name from animals where neutered = false and escape_attempts > 3;
```

Comment:
- [ ] According to the requirements, we need to select name of all animals that are neutered `neutered = true` and have less than 3 escape attempts. `escape_attempts < 3`, kindly update it 💯

![image](https://user-images.githubusercontent.com/45231276/196313040-6ef394b3-e971-401b-be8f-9a8f3017a3c8.png)